### PR TITLE
[minotaur-money] Add minotaur-money strategy

### PR DESCRIPTION
**Pull request by pepperstepper**
Counts up users MINO, sMINO and wsMINO,
in their wallets, as well as in the MMF staking pool.

https://minotaur.money

Fixes # .

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

File: src/strategies/minotaur-money/index.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    MultiCallResult,
+    MultiCallResult,
+    MultiCallObjectResult,
+  ] = await Promise.all([
+    callMinoTotalSupply(),
+    callIndex(),
+    minoMulti.execute(),
+    sMinoMulti.execute(),
+    wsMinoMulti.execute(),
+    wsMinoInMMFMulti.execute()
```
Comment (ChaituVR) [inline src/strategies/minotaur-money/index.ts:97]: Hello @pepperstepper There is a limit of number of requests being sent, should be maximum of 5
https://github.com/snapshotQ-labs/snapshot-strategies#checklist-for-adding-a-new-strategy

Is there some way to reduce number of calls being sent here ?

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/minotaur-money/index.ts (hunk 1 above)
Comment (pepperstepper) [reply src/strategies/minotaur-money/index.ts]: Hi, I have now made it only 5 calls, please re-review :)

File: src/strategies/minotaur-money/index.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { Multicaller, call } from '../../utils';
+
+export const author = 'pepperstepper';
+export const version = '0.0.0';
```
Comment (ChaituVR) [inline src/strategies/minotaur-money/index.ts:5]:
```suggestion
export const version = '0.0.1';
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by ChaituVR** [comment]
I get this error:
```
snapshot-strategies/snapshot-strategies/src/strategies/minotaur-money/index.ts
  23:45  error  Don't use `object` as a type. The `object` type is currently hard to use ([see this issue](https://github.com/microsoft/TypeScript/issues/21732)).
  ```

**Comment by pepperstepper** [comment]
> I get this error:
> 
> ```
> snapshot-strategies/snapshot-strategies/src/strategies/minotaur-money/index.ts
>   23:45  error  Don't use `object` as a type. The `object` type is currently hard to use ([see this issue](https://github.com/microsoft/TypeScript/issues/21732)).
> ```

Hi I changed the object type to any to hopefully resolve this, I don't get this error on my machine.

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**